test/service/test_pagination.py

```diff
@@ -13,6 +12,24 @@ def setUpModule():
     configure_test_logging()
 
 
+internal_pagination_parameters = (
+    'search_after',
+    'search_after_uid',
+    'search_before',
+    'search_before_uid'
+)
+"""
+These pagination parameters are used internally,they are should not be
+included in the `pagination` response field.
```

Comment:
```suggestion
These pagination parameters are used internally, they should not be
included in the `pagination` response field.
```

and rewrap on 80

Reply:
I opted to remove the variable, it was not being used.